frontend/__tests__/doctor/patient-info-modal-content.test.js

```diff
@@ -73,15 +68,43 @@ describe("<PatientInfoModalContent/>", () => {
         expect(wrapper.find(WarningTwoIcon)).toHaveLength(0);
     });
     it("Renders high priority icon if patient is flagged", () => {
-        let w = shallow(<PatientInfoModalContent patient={prioritizedPatient} />);
+        let w = shallow(<PatientInfoModalContent patient={prioritizedPatient} onMutate={jest.fn} onClose={jest.fn} />);
         expect(w.find(WarningTwoIcon)).toHaveLength(1);
     });
     it("Calls modify priority button when priority flag is off", async () => {
-        const component = shallow(<PatientInfoModalContent patient={DEFAULT_PATIENT} />);
+        const component = shallow(
+            <PatientInfoModalContent patient={DEFAULT_PATIENT} onMutate={jest.fn} onClose={jest.fn} />
+        );
         await component.find(Button).at(0).simulate("click");
     });
     it("Calls modify priority button when priority flag is on", async () => {
-        const component = shallow(<PatientInfoModalContent patient={prioritizedPatient} />);
+        const component = shallow(
+            <PatientInfoModalContent patient={prioritizedPatient} onMutate={jest.fn} onClose={jest.fn} />
+        );
         await component.find(Button).at(0).simulate("click");
     });
+    it("Calls modify priority button and return an error", async () => {
+        global.fetch = jest.fn().mockImplementationOnce(() =>
+            Promise.reject({
+                status: 400,
+                json: () => Promise.resolve({ success: false, error: "Something bad happened" }),
+            })
+        );
+
+        const component = shallow(
+            <PatientInfoModalContent patient={DEFAULT_PATIENT} onMutate={jest.fn} onClose={jest.fn} />
+        );
+
+        await component.find(Button).at(0).simulate("click");
+    });
+    it("Calls review all button", async () => {
+        const mutate = jest.fn();
+        const close = jest.fn();
+        const component = shallow(
+            <PatientInfoModalContent patient={prioritizedPatient} onMutate={mutate} onClose={close} />
+        );
+        await component.find("#review-all-button").at(0).simulate("click");
```

Comment:
why is the await needed here when calling the find method?

Reply:
It is needed because a function that is run when the button is click returns a promise.